### PR TITLE
submition_form

### PR DESCRIPTION
**Pull request by latifa-wakili**
in this project is we tried to make a submition form by using html and css

### CODE REVIEW

File: index.html
Comment (ahzamir) [inline index.html:1]: @latifa-wakili 

- Your code is still not clean:
- And for cleaning your code:
  - At the bottom right side of your vscode click on `Spaces` then click on `Index using spaces` and choose 2 from there.
  - Then according to the lines on the screen all child elements should be 2 spaces or one line ahead of their parent element and the closing tag should be in the same line as the opening tag!
  
  - Your body spacing is not aligned correctly:
  - All child elements should be two spaces ahead of their parent element.

**Example:**
```
<div>
  <h1> ... </h1>
  <p>
  .
  .
  </p>
  <div>
    <img ... />
    more elements ...
  </div>
</div>
```

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @latifa-wakili 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Copy your HTML file code and paste it to the freecodecamp website survey form project to see which requirements your project is missing solve those missing requirements and mention me here using @ahzamir to re-review your project.
- You HTML code is not aligned correctly and is not clean please check the inline comments below for more details
# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
❌ You have followed and completed the Survey Form and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

File: index.html
Comment (latifa-wakili) [reply index.html:1]: @ahzamir sir I checked my codes on freecodecamp it passed, there is no any problem
and also I cleaned my html codes now

File: index.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        <div class="coment">
+          <p>Any comments or suggestions?
+          </p>
+          <textarea
+          id="comments"
+          lass="textarea"
+          name="comment"
+          placeholder="Enter your comment here...">
+        </textarea>
+      </textarea>
```
Comment (ahzamir) [inline index.html]: @latifa-wakili 

- Well done now your code passes all checks on freecodecamp, but still you have some issues in this part of your code the closing tag for your 2 divs is not excited here, and besides that you added an extra `</textarea>` closing tag.
- And also your first `</textarea>` is not aligned with its opening tag.
- Solve these mistakes and mention me back here so I can check your project using my username @ahzamir.

**Review by ahzamir — CHANGES_REQUESTED** [review]
@latifa-wakili 

Well done your code passes the checks on freecodecamp but still you have some issues to work on:
- Check the comments below

**Comment by latifa-wakili** [comment]
@ahzamir sir I remove issues you said in  your second feedback 
please check it now

File: index.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+          <p>Any comments or suggestions?
+          </p>
+          <textarea
+          id="comments"
+          lass="textarea"
+          name="comment"
+          placeholder="Enter your comment here...">
+          </textarea>
+        </div><br/>
+        <button type="submit" id="submit" class="submit-button">submit</button>
```
Comment (ahzamir) [inline index.html]: @latifa-wakili 
- These lines are two spaces or one column ahead just use one backspace on each line to solve this error, other than that your code is clean.

**Review by ahzamir — CHANGES_REQUESTED** [review]
@latifa-wakili 
- Check the review

File: index.html (hunk 2 above)
Comment (latifa-wakili) [reply index.html]: @ahzamir I removed extra spcaeses

**Review by ahzamir — APPROVED** [review]
### Hi @latifa-wakili 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Survey Form project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Pull request merged**